js/script.js

```diff
@@ -0,0 +1,3 @@
+$('button').on('click', function(){
```

Comment:
1) Before you add click listener on button, be sure the DOM content has been loaded.
2) Try to achieve the same approach without jQuery. It's not worth to attach entire jQuery library, just to use click listener. You can perform the same functionality using native JavaScript functions.

Reply:
Done. Please check and confirm.